zcash_client_backend/src/encoding.rs

```diff
@@ -279,6 +291,7 @@ mod tests {
     use rand_core::SeedableRng;
     use rand_xorshift::XorShiftRng;
     use zcash_primitives::{
+        constants::{self},
```

Comment:
```suggestion
        constants,
```
I'm surprised that `rustfmt` doesn't pick this up.